main.go

```diff
@@ -33,9 +33,11 @@ const (
 	customizedImagesFile = "images/customized-images.yaml"
 	defaultPlatform      = "linux/amd64"
 	dockerTransport      = "docker://"
-	quayURL              = "quay.io/giantswarm"
-	aliyunURL            = "giantswarm-registry.cn-shanghai.cr.aliyuncs.com/giantswarm"
 	filteredFileSuffix   = ".filtered"
+
+	aliyunURL = "giantswarm-registry.cn-shanghai.cr.aliyuncs.com/giantswarm"
+	azureURL  = "gsoci.azurecr.io"
```

Comment:
Should there be the `/giantswarm` suffix after `gsoci.azurecr.io`?

Reply:
It should! Thank you!